app/templates/demos/_amd.html

```diff
@@ -14,9 +14,9 @@
 	<h1><%= capitalizeName %></h1>
 	<script type="text/javascript">
 		requirejs.config({
-	    baseUrl: '../build/amd'
+			baseUrl: '../../build/amd'
```

Comment:
It's a weird that this would have fixed this demo for you. Adding another `../` should not get the right path, since **demos** is just one folder alway from **build**, not two. I tried this on my side and it wouldn't work for me.

You're right that the current configuration isn't working either though, but from what I saw that's because we're now importing soy files with a `.js` prefix in its path by default, as you can see [here](https://github.com/metal/generator-metal/blob/master/app/templates/src/_BoilerplateComponent.js#L3). We started doing this because otherwise browserify has some problems (it tries to import the **.soy** file instead of the **.soy.js** one). When I run the generator for an amd project and remove the `.js` from the soy import the demo works fine. I'm not sure if there's a good way around this though, maybe only not add `.js` when AMD is chosen?